File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int l = 0;
        int r = numbers.length - 1;
        
        while (r > l) {
            int sum = numbers[l] + numbers[r];
            if (sum > target) {
                r -=1;
            } else if (sum < target) {
                l+=1;
            } else {
                int[] ans = {l+1,r+1};
                return ans;
            }
        }
        return null;
    }
}

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l = 0
        r = len(numbers) - 1
        
        while r > l:
            if (target == numbers[l] + numbers[r]):
                return [l+1,r+1]
            
            if (target > numbers[l] + numbers[r]):
                l+=1
            else:
                r-=1
        